src/test/java/testhelper/PerformanceUtils.java

```diff
@@ -0,0 +1,28 @@
+package testhelper;
+
+import com.sun.management.ThreadMXBean;
+
+import java.lang.management.ManagementFactory;
+
+public class PerformanceUtils {
+
+  private static ThreadMXBean THREAD_MX_BEAN;
```

Comment:
Unfortunately, it's impossible. Field initialized in static block which disallows `final` modifier.